python/test/t_CovarianceModel_std.expout

```diff
@@ -103,21 +103,21 @@ myModel =  SphericalModel(scale=[2], amplitude=[3], radius=4.5)
 active= [0,1]
 parameter= [2,3]
 parameterDescription= [scale_0,amplitude_0]
-myModel( [-1] ,  [3] )= class=SquareMatrix dimension=1 implementation=class=MatrixImplementation name=Unnamed rows=1 columns=1 values=[1.13169]
-dCov = class=Matrix implementation=class=MatrixImplementation name=Unnamed rows=1 columns=1 values=[0.401235]
-dCov (FD)= class=Point name=Unnamed dimension=1 values=[0.401235]
-dCov/dP= [[ 0.8025 ]
- [ 0.3772 ]]
+myModel( [-1] ,  [3] )= class=SquareMatrix dimension=1 implementation=class=MatrixImplementation name=Unnamed rows=1 columns=1 values=[3.39506]
```

Comment:
Since the `.expout` changes so much, shouldn't this work be mentioned as a bugfix in the `ChangeLog`? Users might be surprised if `CovarianceModel`s have different discretizations in 1.17 than in 1.16.

Reply:
In fact the change is due to a bugfix (missing `amplitude`)

Reply:
So users will see no difference between versions?

Reply:
yes.